src/elements/frame-router.ts

```diff
@@ -73,7 +73,8 @@ export default class FrameRouterElement extends HTMLElement {
       hostRootUrl: processedHostUrl
     };
 
-    this.changeRoute(this.getAttribute(ROUTE_ATTR) || 'about:blank');
+    const frameRoute = String(this.getAttribute(ROUTE_ATTR) || '');
```

Comment:
There's not really a good reason to add these empty string fallbacks everywhere _and_ update `getClientTarget` to handle null inputs. What I was getting at attempting is to let the null value of the attribute continue through the call stack, and when it comes time to actually set the frame location , it should fall back to `about:blank` when presented null client data.